recipe/build.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/bash
 
+source activate "${CONDA_DEFAULT_ENV}"
+
 chmod +x configure
 
-./configure --prefix=$PREFIX --enable-cxx
+if [ "$(uname)" == "Darwin" ];
+then
+    ./configure --prefix=$PREFIX --enable-cxx --build=x86_64-apple-darwin
```

Comment:
Fat binaries are to support all types of processor micro-architectures (eg. [nehalem](https://en.wikipedia.org/wiki/Nehalem_(microarchitecture)), [haswell](https://en.wikipedia.org/wiki/Haswell_(microarchitecture)). This is not the difference between 64-bit and 32-bit.

For osx since `--enable-fat` fails, it is compiled using the assembly common to all `x86_64-apple-darwin` processors